.github/workflows/stale.yml

```diff
@@ -0,0 +1,21 @@
+name: Close Stale Pull Requests
+
+on:
+  schedule:
+    #Runs at 00:00 every day
+  - cron: "0 0 * * *"
+
+jobs:
+  stale:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/stale@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-pr-message: 'Hi, were are going to mark this PR as stale because it has not had any updates for 30 days. If no further activity occurs within the following 7 days, it will be automatically closed so that others can take up the issue.
+                            If you are still working on this PR, please make a follow-up commit within 7 days (and submit it for review, if applicable). Please also let us know if you are stuck so we can help you!'
+        stale-pr-label: 'Stale'
```

Comment:
How does this label gets removed when the PR becomes active again?

Reply:
I think needs to be removed manually. Will take a look into this and update.

Reply:
**UPDATE:** The PR will get removed once a comment is added to it. Doesn't work for a commit. Should I update PR message - "Leave a comment if you are still working on this PR?"

Reply:
Sure.
Does this also means a PR is considered stale if there are no comments for 30 days although there are commits within those 30 days?

Reply:
If there are commits, the PR will not be considered stale. But to remove the `stale` label you need to make a comment.